microservices/agreement-readmodel-writer-sql/dev/values.yaml

```
@@ -0,0 +1,53 @@
+name: interop-be-agreement-readmodel-writer-sql
+
+serviceAccount:
+  roleArn: "arn:aws:iam::505630707203:role/interop-be-agreement-readmodel-writer-dev-es1"
+
+service:
+  create: false
+
+configmap:
+  KAFKA_GROUP_ID: "{{.Values.namespace}}-agreement-readmodel-writer-sql"
+  TOPIC_STARTING_OFFSET: "earliest"
+
+deployment:
+  flywayInitContainer:
+    create: true
+    migrationsConfigmap: "flyway-read-model-agreement"
+    env:
+      FLYWAY_CREATE_SCHEMAS: true
+      FLYWAY_PLACEHOLDER_REPLACEMENT: true
+      FLYWAY_PLACEHOLDERS_NAMESPACE: "dev"
```

```suggestion
      FLYWAY_PLACEHOLDERS_NAMESPACE: "{{.Values.namespace}}"
```
we can start using this feature implemented in our chart 😄

---

I can do same [change](https://github.com/pagopa/interop-core-deployment/blob/main/microservices/attribute-registry-readmodel-writer-sql/dev/values.yaml#L20) in `attribute-registry-rmw-sql`. OK?

---

I can do same [change](https://github.com/pagopa/interop-core-deployment/blob/main/microservices/attribute-registry-readmodel-writer-sql/dev/values.yaml#L20) in `attribute-registry-rmw-sql`. OK?

---

Yes 👍

---

Done